### DISCUSSION TITLE
Connect 2 network taps to one heavy node?

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#9694] [Unsupported Versions]
Hello!

assuming I have enough network interfaces, can I connect two network taps to one heavy node in order to monitor traffic?

Cheers, Ben

**Comment by reyesj2** [comment]
Yes, just add the additional network tap to another port on the heavy node configured as a monitor port ```so-monitor-add```

I would also err on the side of caution and ensure your heavy nodes have good enough hardware to process the increased data.
